### PR TITLE
Spell check

### PR DESCRIPTION
**Pull request by fahurox**
I have fixed a couple of spelling mistakes and improved some grammar in these testimonial files

### CODE REVIEW

File: docs/about/case-studies/puppet.md [hunk 1]
```diff
@@ -36,36 +36,36 @@ As their target came into focus, they realized that the serverless applications
...
 
 <h2 style="color:#0071c7;"> Workflow integrations</h2>
 
-The open architecture allows Relay to integrate dozens of different services and platforms into workflows. A look at the <a href="https://github.com/relay-integrations">Relay integrations GitHub page</a> provides a list of these integrations and demonstrates their commitment to the open source community.
+The open architecture allows Relay to integrate dozens of different services and platforms into workflows. A look at the <a href="https://github.com/relay-integrations">Relay integrations GitHub page</a> provides a list of these integrations and demonstrates their commitment to the open-source community.
 
-"I'm a strong believer in working with open source projects. We've made contributions to numerous projects, including Tekton, Knative, Ambassador, and gVisor, all of which we depend on to make our product functional," says Fontes.
+"I'm a strong believer in working with open-source projects. We've made contributions to numerous projects, including Tekton, Knative, Ambassador, and gVisor, all of which we depend on to make our product functional," says Fontes.
```
Comment (abrennan89) [inline docs/about/case-studies/puppet.md:56]: same comment re "open source"

File: docs/about/case-studies/puppet.md [hunk 2]
```diff
@@ -36,36 +36,36 @@ As their target came into focus, they realized that the serverless applications
...
 
 <h2 style="color:#0071c7;"> Results: automated infrastructure management</h2>
 
-While Relay's infrastructure runs on Google Cloud Platform, its <a href="https://relay.sh/library/">library of workflows, integrations, steps, and triggers</a> includes services across all major cloud service providers. Relay customers can integrate across Microsoft Azure, AWS, and Oracle Cloud Infrastructure among others. By combining these integrations with SaaS offerings, it truly is becoming the <a href=https://zapier.com/">Zapier</a> of infrastructure management.
+While Relay's infrastructure runs on the Google Cloud Platform, its <a href="https://relay.sh/library/">library of workflows, integrations, steps, and triggers</a> includes services across all major cloud service providers. Relay customers can integrate across Microsoft Azure, AWS, and Oracle Cloud Infrastructure among others. By combining these integrations with SaaS offerings, it truly is becoming the <a href=https://zapier.com/">Zapier</a> of infrastructure management.
```
Comment (abrennan89) [inline docs/about/case-studies/puppet.md:60]:
```suggestion
While Relay's infrastructure runs on the Google Cloud Platform, it is a <a href="https://relay.sh/library/">library of workflows, integrations, steps, and triggers</a> that includes services across all major cloud service providers. Relay customers can integrate across Microsoft Azure, AWS, and Oracle Cloud Infrastructure, among others. By combining these integrations with SaaS offerings, it truly is becoming the <a href=https://zapier.com/">Zapier</a> of infrastructure management.
```

File: docs/about/case-studies/puppet.md [hunk 3]
```diff
@@ -36,36 +36,36 @@ As their target came into focus, they realized that the serverless applications
...
 
 Building and deploying Relay would not have been possible without the existing infrastructure offered by systems such as Knative and <a href="https://tekton.dev">Tekton</a>. Remarkably, Fontes' team never grew above eight engineers. Once they solidified their plan for Relay, they were able to bring it to production in just three months, says Fontes.
 
 <h2 style="color:#0071c7;"><em>"Thanks to Knative, getting Relay out the door was easier than we thought it would be."</em> said Noah Fontes, Senior Principal Software Engineer.</h2>
 
-Knative aims to make scalable, secure, stateless architectures available quickly by abstracting away the complex details of a Kubernetes installation and enabling developers to focus on what matters.
+Knative aims to make scalable, secure, stateless architectures to be available quickly by abstracting away the complex details of a Kubernetes installation and enabling developers to focus on what matters.
```
Comment (abrennan89) [inline docs/about/case-studies/puppet.md:68]: this was correct before, please revert

File: docs/about/case-studies/outfit7.md [hunk 4]
```diff
@@ -31,25 +31,25 @@ The success of the well-known game developer behind massive hits like the Talkin
 
 With up to 470 million monthly active users, 20 thousand server requests per second, and terabytes of data generated daily, they needed a stable, high-performance solution. They turned to a Knative and Kubernetes solution to optimize real-time bidding ad sales in a way that can automatically scale up and down as needed.  They were able to develop a system so easy to maintain that it freed up two software engineers who are now able to work on more important tasks, like optimizing backend costs and adding new game features.
 
-<h2 style="color:#0071c7;">High performance in-app bidding</h2>
+<h2 style="color:#0071c7;">High-performance in-app bidding</h2>
```
Comment (abrennan89) [inline docs/about/case-studies/outfit7.md:34]: doesn't need a hyphen, please revert to "high performance"

**Review by abrennan89 — APPROVED** [review]
/lgtm

**Comment by psschwei** [comment]
/hold
for CLA

**Comment by psschwei** [comment]
@fahurox please let us know when you've signed the CLA so we can unhold this

**Comment by fahurox** [comment]
I have signed the CLA

**Pull request merged**